packages/components/src/Combobox/components/ComboboxContent/ComboboxContentList/ComboboxContentList.tsx

```diff
@@ -1,49 +1,21 @@
 import React, { useEffect, useState } from "react";
 import classnames from "classnames";
 import { Text } from "@jobber/components/Text";
+import { Glimmer } from "@jobber/components/Glimmer";
 import styles from "./ComboboxContentList.css";
-import { ComboboxListProps } from "../../../Combobox.types";
+import {
+  ComboboxListProps,
+  ComboboxOption as ComboboxOptionType,
+} from "../../../Combobox.types";
 import { ComboboxOption } from "../../ComboboxOption/ComboboxOption";
 
 export function ComboboxContentList(props: ComboboxListProps): JSX.Element {
-  const [listScrollState, setlistScrollState] = useState("");
-
-  useEffect(() => {
```

Comment:
moving all this into a local hook isn't necessary

I was in here and felt like the effect hook was taking up a considerable amount of line real estate in this component and wanted to tidy that up a bit